qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -378,85 +378,107 @@
     },
     {
       "tabTitle": "Summary",
-      "charts": {},
-      "filters": {
-        "tabs": [{
-            "title": "Reports",
-            "fields": [
-              "staffed_all_beds",
-              "staffed_icu_beds",
-              "licensed_all_beds",
-              "all_bed_occupancy_rate",
-              "icu_bed_occupancy_rate",
-              "population",
-              "population_gtr_20",
-              "population_gtr_65",
-              "estimated_full_mech_ventilators",
-              "estimated_full_mech_pediatric_ventilators"
-            ]
-          },
-          {
-            "title": "Location",
-            "fields": [
-              "county",
-              "province_state"
-            ]
-          }
-        ]
-      },
-      "table": {
-        "enabled": false,
-        "fields": []
+      "charts": {
+        "country_region": {
+          "chartType": "pie",
+          "title": "Country"
+        }
       },
-      "dropdowns": {},
-      "buttons": [],
-      "guppyConfig": {
-        "dataType": "location",
-        "fieldMapping": [{
-            "field": "staffed_all_beds",
-            "name": "All Staffed Beds"
-          },
-          {
-            "field": "staffed_icu_beds",
-            "name": "All ICU Beds"
-          },
-          {
-            "field": "licensed_all_beds",
-            "name": "All Licensed Beds"
-          },
-          {
-            "field": "icu_bed_occupancy_rate",
-            "name": "ICU Beds Occupancy Rate"
-          },
-          {
-            "field": "population_gtr_20",
-            "name": "Population Greater Than 20"
-          },
-          {
-            "field": "population_gtr_65",
-            "name": "Population Greater Than 65"
-          },
-          {
-            "field": "estimated_full_mech_ventilators",
-            "name": "Estimated Ventilators"
-          },
-          {
-            "field": "estimated_full_mech_pediatric_ventilators",
-            "name": "Estimated Pediatric Ventilators"
-          },
-          {
-            "field": "province_state",
-            "name": "Province or State"
+        "filters": {
+          "tabs": [
+            {
+              "title": "Reports",
+              "fields": [
+                "staffed_all_beds",
+                "staffed_icu_beds",
+                "licensed_all_beds",
+                "all_bed_occupancy_rate",
+                "icu_bed_occupancy_rate",
+                "population",
+                "population_gtr_20",
+                "population_gtr_65",
+                "estimated_full_mech_ventilators",
+                "estimated_full_mech_pediatric_ventilators"
+              ]
+            },
+            {
+              "title": "Location",
```

Comment:
i would put this filter tab in first position, and the "Reports" tab in second position